internal/core/ports/wallet.go

```diff
@@ -142,7 +135,6 @@ type TxNotification interface {
 }
 
 type UtxoNotification interface {
-	AccountIndex() uint64
```

Comment:
Why did you remove this?

Reply:
I think because u never use it but I can revert it. It raises a question: the `ports` interfaces should really implements ALL the proto types members ? even if we are not gonna use it inside TDEX services ?

Reply:
I will use it for settling or expring trades actually, so yes, please, revert the change.